doc/design/upgrade.md

```diff
@@ -38,14 +59,16 @@ If anything wrong happens during upgrade, vic-machine will rollback VCH to origi
 
 ### Downgrade
 
-For any running VCH, VIC admin can downgrade existing VCH to an old version. (I assume this function is only used for upgrade roll back, we might not want to recommend downgrade for anything working well, then the downgrade scope will be same to upgrade versions)
-
-Verified docker-engine upgrade/downgrade, which works well between 1.11.2 and 1.12. The running container will be stopped after upgrade/downgrade, but is good to start again in new version. And all images and containers information is not lost. So from technical point of view, we should support same story.
+Verified docker-engine upgrade/downgrade, which works well between 1.11.2 and 1.12. The running container will be stopped after upgrade/downgrade, but is good to start again in new version. And all images and containers information is not lost. But think of the complexity to support version downgrade, we'll not go with this at this moment, unless there is explicity user requirement.
```

Comment:
So, what do you mean for roll-back of one update? I assume we don't have roll-back at all. What we did is to roll back only when upgrade failed.